packages/ephemeral/indexer.js

```diff
@@ -5,11 +5,7 @@ module.exports = declareInjections({
   loadInitialModels: 'config:initial-models'
 }, class Indexer {
 
-  async branches() {
-    return ['master'];
-  }
-
-  async beginUpdate(/* branch */) {
+  async beginUpdate() {
```

Comment:
@robustdj additionally, no more need to pass in the `branch` parameter in the `beginUpdate()` of any indexers that you encounter in the cards or portfolio.